### DISCUSSION TITLE
P1 - GND, VIN - specification?

### DISCUSSION THREAD

**Discussion by mvschaefer** [witnessmenow/ESP32-Cheap-Yellow-Display#146] [Q&A]
I am considering using the P1 port to provide power externally.

What are the Vmin / Vmax values that are allowable in this use case?

Thanks.

**Comment by DoctorU** [comment]
First, my legal protections: Caveat Emptor;  Your Mileage May Vary(tm). If you blow up your board, that's on you! 😉  

It doesn't look like the board has any special voltage protection.
It seems to be designed for a standard USB 5V input ( +-5% = 4.75V to 5.25).
If you want to use higher input voltage, then put a step-down circuit inline with VIN.
The 5V input is then stepped down (by the AMS1117-3.3) to 3.3v for most of the ciruitry. See "Section: 3.3V-ESP32" below.

A LiPo battery can be attached for backup operation, which supplies around 3.7V into the 3.3V voltage rail.


---
# Schematic 
I started looking here: https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/blob/main/OriginalDocumentation/5-Schematic/ESP32-2432S028-MCU.jpg

## Section: 3.3V-ESP32
The board expects a 5V input voltage.
Power to the 3.3V rail is stepped down from a 5V input by an AMS1117-3.3.
ABSOLUTE MAXIMUM INPUT VOLTAGE ON THIS IC is 15V; And it looks like only P1 and the USB1 port are attached to the 5V rail.
So if you supply more than 5V on P1 AND connect to USB1, you will likely break your PC/connected device.
[ds1117.pdf](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/files/15060100/ds1117.pdf)

## Section: POWER

The VIN is regulated by a AO3401 FT, which has a maximum GS voltage of 12V!
[AO3401.pdf](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/files/15059812/AO3401.pdf)

## Section: USB 2 TTL
There's no input voltage limiting. 

Component D1, which is a 1N5819W Shottky Diode, limits output current to the USB port to 0.1mA when power is provided from P1. 
[1N5817W THRU 1N5819W N1756 REV.A.pdf](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/files/15060024/1N5817W.THRU.1N5819W.N1756.REV.A.pdf)
